docs/dev/table/connectors/formats/csv.zh.md

```diff
@@ -0,0 +1,330 @@
+      <td><h5>line-delimiter</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;"><code>\n</code></td>
+      <td>String</td>
+      <td>Line delimiter ('\n' by default, otherwise
+      '\r' or '\r\n' are allowed), unicode is supported if
+      the delimiter is an invisible special character,
+      e.g. U&'\\000D' is the unicode representation of carriage return '\r'
+      e.g. U&'\\000A' is the unicode representation of line feed '\n'.</td>
+    </tr>
+    <tr>
+      <td><h5>disable-quote-character</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Flag to disabled quote character for enclosing field values (false by default)
+      if true, quote-character can not be set.</td>
+    </tr>
+    <tr>
+      <td><h5>quote-character</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;"><code>"</code></td>
+      <td>String</td>
+      <td>Quote character for enclosing field values ('"' by default).</td>
+    </tr>
+    <tr>
+      <td><h5>allow-comments</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Flag to ignore comment lines that start with '#'
+      (disabled by default);
+      if enabled, make sure to also ignore parse errors to allow empty rows.</td>
+    </tr>
+    <tr>
+      <td><h5>ignore-parse-errors</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">false</td>
+      <td>Boolean</td>
+      <td>Flag to skip fields and rows with parse errors instead of failing;
+      fields are set to null in case of errors.</td>
+    </tr>
+    <tr>
+      <td><h5>array-element-delimiter</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;"><code>;</code></td>
+      <td>String</td>
+      <td>Array element delimiter string for separating
+      array and row element values (';' by default).</td>
+    </tr>
+    <tr>
+      <td><h5>escape-character</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Escape character for escaping values (disabled by default).</td>
+    </tr>
+    <tr>
+      <td><h5>null-literal</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Null literal string that is interpreted as a
+      null value (disabled by default).</td>
+    </tr>
+    </tbody>
+</table>
+
+Data Type Mapping
+----------------
+
+Currently, the CSV schema is always derived from table schema. Explicitly defining an CSV schema is not supported yet.
+
+Flink CSV format uses utility class `com.fasterxml.jackson.databind.node.ObjectNode` (for serialization) and `com.fasterxml.jackson.databind.JsonNode` (for deserialization) provided by `jackson` to convert Flink Data Types to and from CSV text.
+
+The following table lists the type mapping from Flink type to CSV type.
```

Comment:
I think we can copy the data type mapping of JSON to here. They are almost the same (except CSV doesn't some types, e.g. MAP).